google/cloud/aiplatform/metadata/metadata.py

```diff
@@ -94,11 +119,68 @@ def log_metrics(self, metrics: Dict[str, Union[str, float, int]]):
         )
         artifact.update(metadata=metrics)
 
-    def get_experiment(self, experiment: str):
-        raise NotImplementedError("get_experiment not implemented")
+    def get_experiment(self, experiment: str) -> pd.DataFrame:
+        """Returns a Pandas DataFrame of the parameters and metrics associated with one experiment.
+
+            Example:
+
+            aiplatform.init(experiment='exp-1')
+            aiplatform.start_run(run='run-1')
+            aiplatform.log_params({'learning_rate': 0.1})
+            aiplatform.log_metrics({'accuracy': 0.9})
+
+            aiplatform.start_run(run='run-2')
+            aiplatform.log_params({'learning_rate': 0.2})
+            aiplatform.log_metrics({'accuracy': 0.95})
+
+            Will result in the following DataFrame
+            ___________________________________________________________________________
+            | experiment_name | run_name      | param.learning_rate | metric.accuracy |
+            ---------------------------------------------------------------------------
+            | exp-1           | run-1         | 0.1                 | 0.9             |
+            | exp-1           | run-2         | 0.2                 | 0.95            |
+            ---------------------------------------------------------------------------
+
+            Args:
+              experiment: Name of the Experiment to filter results.
+
+            Returns:
+              Pandas Dataframe of Experiment with metrics and parameters.
+            """
+
+        source = "experiment"
+        experiment_resource_name = self._get_experiment_or_pipeline_resource_name(
+            name=experiment, source=source, expected_schema=constants.SYSTEM_EXPERIMENT,
+        )
+
+        if experiment_resource_name:
```

Comment:
It looks like this Optionally returns an Experiment. I think we should throw if the experiment is not found. Same comment for Pipelines below.

Reply:
done.